components/elm/src/biogeophys/HydrologyDrainageMod.F90

```diff
@@ -147,10 +147,10 @@ subroutine HydrologyDrainage(bounds,    &
 
 #ifndef _OPENACC
       if (use_betr) then
-        call ep_betr%BeTRSetBiophysForcing(bounds, col_pp, veg_pp, 1, nlevsoi, waterstate_vars=waterstate_vars, &
-          waterflux_vars=waterflux_vars)
+        call ep_betr%BeTRSetBiophysForcing(bounds, col_pp, veg_pp, 1, nlevsoi, waterstate_vars=col_ws, &
+          waterflux_vars=col_wf)
```

Comment:
If the waterflux_vars and waterstate_vars were only used by sbetr before, please remove their modules in this module, i.e. remove the following statements:
  use WaterStateType, only : waterstate_vars
  use WaterFluxType , only : waterflux_vars

Reply:
Changes made.